xdsl/transforms/experimental/ConvertStencilToLLMLIR.py

```diff
@@ -44,7 +44,7 @@
 def GetMemRefFromField(
     input_type: FieldType[_TypeElement] | TempType[_TypeElement],
 ) -> MemRefType[_TypeElement]:
-    dims = [i.value.data for i in input_type.shape.data]
+    dims = [i for i in input_type.shape]
```

Comment:
```suggestion
    dims = tuple(input_type.shape)
```

Reply:
I'm going for simply input_type.shape as an argument, as it is an Iterable of int now, thanks for catching those though!

Reply:
I'm confused, is this an upcoming change in the diff?